src/%IPM/Common.inc

```diff
@@ -6,10 +6,10 @@ ROUTINE %IPM.Common [Type=INC]
 #define ZPMStudioDocumentModule %ZPMStudioDocumentModule
 
 #; Local % Variable to track modules already handled for a given end goal
-#define ZPMHandledModules %ZZPMHandledModules
+#define ZPMHandledModules %ZPMHandledModules
```

Comment:
The values here can all be changed to IPM instead of ZPM and IRIS.Temp instead of CacheTemp (the names of the macros can remain as ZPM to avoid extra work of replacing all those references for now).